core/trino-web-ui/src/main/resources/webapp-preview/package.json

```diff
@@ -11,19 +11,25 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@douyinfe/semi-icons": "^2.63.1",
```

Comment:
I think that we should rather use https://www.npmjs.com/package/material-symbols


cc @mosabua

Reply:
let’s not mix items from two different design packages. (semi and material). If You prefer material look and feel then let’s switch not only the icons but the entire lib, otherwise we might end up with a weird and mixed look and feel.

Reply:
yeah! makes sense :)

Reply:
@koszti is this a heavy lift to use material ui here?

Reply:
so material ui + material symbols (as called out here: https://m3.material.io/styles/icons/overview)